release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2396,6 +2396,116 @@ $ ./openshift-install wait-for install-complete
 
 * The `loadBalancerSourceRanges` attribute is not supported, and is thus ignored, in load-balancer type services in clusters that run on {rh-openstack} and use the OVN Octavia provider. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-2789[*OCPBUGS-2789*]) 
 
+* After a catalog source update, it takes time for OLM to update the subscription status.
+This can mean that if you modify a catalog source policy in a `ClusterGroupUpdate` CR, a subscription policy in the same `ClusterGroupUpdate` CR is not remediated as its status has not had sufficient time to update and continues to show it as compliant.
+As a workaround, include a status check in the catalog source policy in the `spec` section.
```

Comment:
s/include a status check in the catalog source policy in the `spec` section/include a status check in the `spec` section of the catalog source policy.

Reply:
When do you perform the workaround?